### DISCUSSION TITLE
Customizing dropdown menu example to act same as OS menu system

### DISCUSSION THREAD

**Discussion by tounsoo** [floating-ui/floating-ui#2461] [Q&A] [answered]
On MacOS and Windows dropdown submenu doesn't close when hovering out of the menu tree. 

With the current dropdown menu example on the site ([code sandbox](https://codesandbox.io/s/admiring-lamport-5wt3yg?file=/src/DropdownMenu.tsx)), I'm trying to disable closing currently open submenu(s) when user hovers outside of the tree.

I feel like there should be a way of controlling `useHover` or `useDismiss` to do this but I'm totally stuck. 

Help!

**Comment by atomiks** [comment, marked as answer]
https://github.com/floating-ui/floating-ui/issues/2446#issuecomment-1636640897

**Reply by tounsoo** [reply]
Thank you for the info! 

For those who are also looking for the same thing, I made a [code sandbox](https://codesandbox.io/s/dropdownmenu-floating-ui-zl8lp4) with the full code.
